themes/bootstrap3/js/common.js

```diff
@@ -66,19 +66,17 @@ var VuFind = (function VuFind() {
   };
 
   var initDisableSubmitOnClick = function initDisableSubmitOnClick() {
-    $('[data-disable-on-submit]').on('submit', function handleOnClickDisable() {
-      var $form = $(this);
+    var form = document.querySelector("[data-disable-on-submit]");
```

Comment:
This looks a lot better but we need to apply this to every matching form on the patch, so this line needs to be replaced with a loop like:

```js
document.querySelectorAll("[data-disable-on-submit]").forEach((form) => {
// the eventlistener and setTimeout can stay the same
});
```